examples/download_sample_data.py

```diff
@@ -24,23 +24,28 @@
 This module provides the progressbar functionality.
 
 """
+from collections import OrderedDict
+from os import path
+import glob
+import os
+import subprocess
 import sys
-import time
 import tarfile
+import time
 import zipfile
 
-import os
 import yaml
-
 try:
     import requests
 except ImportError:
     print('this download script requires the requests module: conda install requests')
     sys.exit(1)
 
-from collections import OrderedDict
-
-from os import path
+try:
+    from py7zlib import Archive7z
+except:
+    subprocess.check_output(['pip', 'install', 'pylzma'], env=os.environ)
+    from py7zlib import Archive7z
```

Comment:
I'm a bit uncomfortable modifying the user's environment behind the scenes in this way.

I would favor adding pylzma to environment.yml instead, and then there testing if it's installed, and just skipping the lidar example data altogether if it's not.